src/sphinx/4.0.x/features/scaladebugger/expression-evaluator.rst

```diff
@@ -0,0 +1,154 @@
+.. Copyright (c) 2015 Contributor. All rights reserved.
+
+.. include:: /global_defs.hrst
+
+.. _expression-evaluator-user-docs:
+
+Expression evaluator user documentation
+=======================================
+
+Scala debugger uses new expression evaluator, which translates user code into invocations of ``JDI`` remote calls. This
+allows evaluation of expression in context of some breakpoint, with access to local variables and methods.
+
+Expression evaluator contains of separate view (similar to ``Scala interpreter`` view) and hook for conditional
+breakpoints.
+
+Evaluator view
+--------------
+
+To open evaluator view use ``Window → Show View → Other → Debug → Scala Expression Evaluator``. In there, any
+expression can be evaluated in context of your current breakpoint. This allows usage any values and types visible
+from breakpoint and (possibly) modification of state of your application.
+
+.. image:: images/expression-evaluator/expression.png
+   :alt: Sample expression evaluation
```

Comment:
Some of these images look bad when they are rendered, they are too wide. Add `:width: 100%` as a tag to inline them in the layout.